src/common/options.cc

```diff
@@ -7853,6 +7853,35 @@ static std::vector<Option> get_immutable_object_cache_options() {
     Option("immutable_object_cache_watermark", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
     .set_default(0.1)
     .set_description("immutable object cache water mark"),
+
+    Option("immutable_object_cache_qos_schedule_tick_min", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
```

Comment:
Nit: there apparently is a new `TYPE_SECS` that probably should be used for this new code (applies to `XYZ_seconds` options as well)

Reply:
> Nit: there apparently is a new `TYPE_SECS` that probably should be used for this new code (applies to `XYZ_seconds` options as well)

But in throttle.cc, following 2 functions still use "uint64", if I change to TYPE_SECS and change val type to "secs", I still need to cast to "uint64" otherwise it will report an error.
_void TokenBucketThrottle::set_schedule_tick_min(uint64_t tick)
int TokenBucketThrottle::set_limit(uint64_t average, uint64_t burst, uint64_t burst_seconds)_

Reply:
> You can either change `TokenBucketThrottle` to use the interval or cast it before sending it to `TokenBucketThrottle` -- but either way, we should be using the most appropriate types in the config.

I have change burst seconds type to TYPE_SECS, but "qos_schedule_tick_min" is in millisecond. There no TYPE_MILLISECS, and also can't use float in TYPE_SECS. I didn't change it.  How about "qos_schedule_tick_min".

Reply:
> You can either change `TokenBucketThrottle` to use the interval or cast it before sending it to `TokenBucketThrottle` -- but either way, we should be using the most appropriate types in the config.

I have change burst second to TYPE_SECS, but "qos_schedule_tick_min" is in milliseconds. There is no TYPE_MILLISECS. How about it

Reply:
I have change burst second to TYPE_SECS, but "qos_schedule_tick_min" is in milliseconds. There is no TYPE_MILLISECS. How about it

Reply:
> You can either change `TokenBucketThrottle` to use the interval or cast it before sending it to `TokenBucketThrottle` -- but either way, we should be using the most appropriate types in the config.

I have change burst second to TYPE_SECS, but "qos_schedule_tick_min" is in milliseconds. There is no TYPE_MILLISECS. How about it

Reply:
Sure, makes sense to me.